bazel/browsers/chromium/chromium.bzl

```diff
@@ -41,6 +40,10 @@ def define_chromium_repositories():
         named_files = {
             "CHROMIUM": "chrome-mac/Chromium.app/Contents/MacOS/Chromium",
         },
+        exclude_patterns = [
+            # Exclude a log file that chromium writes to each run, causing remote cache misses.
+            "chrome-mac/Chromium.app/Contents/Frameworks/Chromium Framework.framework/Versions/*/chrome_debug.log",
```

Comment:
Log gets output under `Versions/Current` and `Versions/99.0.4759.0`.

Reply:
I don't actually have an `x64` mac to test this on, but I'm assuming the path is the same as the `arm64`.